nemo/collections/tts/torch/data.py

```diff
@@ -180,6 +180,7 @@ def __init__(
                         "mel_filepath": item["mel_filepath"] if "mel_filepath" in item else None,
                         "duration": item["duration"] if "duration" in item else None,
                         "speaker_id": item["speaker"] if "speaker" in item else None,
+                        "phoneme": item["phoneme"] if "phoneme" in item else None,
```

Comment:
> Alternatively (and preferably), if this is consistent across the entire dataset, can you move this to be a parameter for the Dataset class instead of per-sample, add a phonemes field with the phonemized text (instead of putting it in normalized_text) for each, then duplicate each sample when loading? Then it can be set in the config instead of having to modify the manifest.

I agree. Let me make a separate PR for next release to make this correction.

Meanwhile,
> Can you rename this to something more descriptive? I think phoneme (or similar) intuitively should be the text converted to phonemes rather than an indicator.

how about `is_phoneme`?

Reply:
Would it be feasible to roll it into this PR? I think we should try to avoid introducing and immediately breaking a new feature of the data loader within one release cycle.

Reply:
Per our discussions, will take this up in the next release.

cc: @XuesongYang